plus/src/main/java/org/apache/calcite/adapter/os/OsQueryTableUtil.java

```diff
@@ -0,0 +1,386 @@
+      if (NetFlags.LOOPBACK_ADDRESS.equals(cfg.getAddress())
+          || (cfg.getFlags() & NetFlags.IFF_LOOPBACK) != 0
+          || NetFlags.NULL_HWADDR.equals(cfg.getHwaddr())) {
+        continue;
+      }
+      objects[0] = cfg.getAddress();
+      objects[1] = cfg.getBroadcast();
+      objects[2] = cfg.getHwaddr();
+      objects[3] = cfg.getNetmask();
+      objects[4] = cfg.getDescription();
+      objects[5] = cfg.getType();
+      list.add(objects);
+    }
+    return list;
+  }
+
+  private List<Object[]> mounts() throws Exception {
+    final List<Object[]> list = new ArrayList<Object[]>();
+    final FileSystem[] fslist = sigar.getFileSystemList();
+    for (int i = 0; i < fslist.length; i++) {
+      Object[] objects = new Object[14];
+      FileSystem fs = fslist[i];
+      FileSystemUsage usage = null;
+      usage = sigar.getFileSystemUsage(fs.getDirName());
+      objects[0] = i;
+      objects[1] = fs.getDevName();
+      objects[2] = fs.getDirName();
+      objects[3] = fs.getFlags();
+      objects[4] = fs.getSysTypeName();
+      objects[5] = fs.getTypeName();
+      objects[6] = fs.getType();
+      objects[7] = usage.getDiskReads();
+      objects[8] = usage.getDiskWrites();
+      switch (fs.getType()) {
+      // TYPE_UNKNOWN ：unknown
+      case 0:
+        break;
+      // TYPE_NONE
+      case 1:
+      case 2:
+        double usePercent = usage.getUsePercent() * 100D;
+        objects[9] = HexTransformationUtils.kbToGb(usage.getTotal()) + "GB";
+        objects[10] = HexTransformationUtils.kbToGb(usage.getFree()) + "GB";
+        objects[11] = HexTransformationUtils.kbToGb(usage.getAvail()) + "GB";
+        objects[12] = HexTransformationUtils.kbToGb(usage.getUsed()) + "GB";
+        objects[13] = usePercent + "%";
+        break;
+      // TYPE_NETWORK ：internet
+      case 3:
+        break;
+      // TYPE_RAM_DISK ：Flash memory
+      case 4:
+        break;
+      // TYPE_CDROM ：Optical drive
+      case 5:
+        break;
+      // TYPE_SWAP ：Page exchang
+      case 6:
+        break;
+      default:
+        break;
+      }
+      list.add(objects);
+    }
+    return list;
+  }
+
+  private List<Object[]> whoUse() throws SigarException {
+    final List<Object[]> list = new ArrayList<Object[]>();
+    final Who[] whos = sigar.getWhoList();
+    if (whos != null && whos.length > 0) {
+      for (Who who : whos) {
+        Object[] objects = new Object[4];
+        objects[0] = who.getDevice();
+        objects[1] = who.getHost();
+        objects[2] = who.getUser();
+        objects[3] = who.getTime();
+        list.add(objects);
+      }
+    }
+    return list;
+  }
+
+  private void loadLibs() {
+    if (StringUtils.isEmpty(System.getProperty("org.hyperic.sigar.path"))) {
+      String sigarPathStr = "/hyperic-sigar-1.6.4/sigar-bin/lib/";
+      String prop = System.getProperty("org.hyperic.sigar.path", sigarPathStr);
+      URL sigarPathUrl = OsQueryTableUtil.class.getResource(prop);
```

Comment:
`System.getProperty("org.hyperic.sigar.path")` is already empty, so we can return `sigarPathStr` directly.
Also, can we make these string constants `"org.hyperic.sigar.path"` and `"/hyperic-sigar-1.6.4/sigar-bin/lib/"` a static final member ?